omod/src/test/java/org/openmrs/module/owa/web/controller/OwaRestControllerTest.java

```diff
@@ -0,0 +1,102 @@
+package org.openmrs.module.owa.web.controller;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.net.URL;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.openmrs.GlobalProperty;
+import org.openmrs.api.context.Context;
+import org.openmrs.api.context.ServiceContext;
+import org.openmrs.messagesource.MessageSourceService;
+import org.openmrs.util.OpenmrsClassLoader;
+import org.openmrs.web.WebConstants;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.mock.web.MockServletContext;
+
+public class OWARestControllerTest extends BaseModuleWebContextSensitiveTest{

```

Comment:
Your code will not compile because your class and constructor name is "OWARestControllerTest" and you have created file with name "OwaRestControllerTest.java". In java both file name and class name should be same. So please change the name of class and constructor to "OwaRestControllerTest".